File: source/includes/fts/search-index-management/create-indexes.java
import com.mongodb.client.MongoClient;
import com.mongodb.client.MongoClients;
import com.mongodb.client.MongoCollection;
import com.mongodb.client.MongoDatabase;
import com.mongodb.client.model.SearchIndexModel;
import org.bson.Document;
import java.util.Arrays;

public class CreateIndex {
    public static void main(String[] args) {
        // connect to your Atlas cluster
        String uri = "<connection-string>";

        try (MongoClient mongoClient = MongoClients.create(uri)) {
            // set namespace
            MongoDatabase database = mongoClient.getDatabase("<databaseName>");
            MongoCollection<Document> collection = database.getCollection("<collectionName>");

            SearchIndexModel indexOne = new SearchIndexModel("<first-index-name>",
                    new Document( 
                        // search index definition fields
                    ));

            SearchIndexModel indexTwo = new SearchIndexModel("<second-index-name>",
                    new Document( 
                        // search index definition fields
                    ));

            collection.createSearchIndexes(Arrays.asList(indexOne, indexTwo));
        }
    }
}

File: source/includes/fts/search-index-management/csharp/CreateIndexes.cs
using MongoDB.Bson;
using MongoDB.Driver;

// connect to your Atlas deployment
var uri = "<connection-string>";

var client = new MongoClient(uri);

var db = client.GetDatabase("<databaseName>");
var collection = db.GetCollection<BsonDocument>("<collectionName>");

// define your Atlas Search indexes
var indexes = new List<CreateSearchIndexModel>
{
    new CreateSearchIndexModel(
        "<first-index-name>",
        new BsonDocument
        {
            // search index definition fields
        }
    ),
    ...
    new CreateSearchIndexModel(
        "<last-index-name>",
        new BsonDocument
        {
            // search index definition fields
        }
    )
};

var result = collection.SearchIndexes.CreateMany(indexes);
